scripts/dts/extract/globals.py

```diff
@@ -331,6 +332,30 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def gpio_map_lookup(cell_parent, pin_ref):
+    index = 0
+    map_array = []
+
+    raw_map = reduced[cell_parent]['props'].get('gpio-map')
```

Comment:
Using `.get('gpio-map')` instead of `['gpio-map']` here isn't safer, because if `raw_map` ever ends up as `None`, then `len(raw_map)` below will crash.

Reply:
Ok, so using ['gpio-map'], then.
This should not end up as `None` since function is called only if `gpio-map` in `reduced...`.